ext/jinja2/src/main/java/org/eclipse/krazo/ext/jinja2/Jinja2ViewEngine.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2021 Eclipse Krazo committers and contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.eclipse.krazo.ext.jinja2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.charset.Charset;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.hubspot.jinjava.Jinjava;
+import com.hubspot.jinjava.interpret.RenderResult;
+import com.hubspot.jinjava.interpret.TemplateError;
+
+import org.eclipse.krazo.engine.ViewEngineBase;
+
+import jakarta.annotation.Priority;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.mvc.engine.ViewEngine;
+import jakarta.mvc.engine.ViewEngineContext;
+import jakarta.mvc.engine.ViewEngineException;
+import jakarta.servlet.ServletContext;
+
+/**
+ * Class Jinja2ViewEngine.
+ *
+ * @author Jeremias Weber
+ */
+@ApplicationScoped
+@Priority(ViewEngine.PRIORITY_FRAMEWORK)
+public class Jinja2ViewEngine extends ViewEngineBase {
+
+    protected final Jinjava jinjava = new Jinjava();
```

Comment:
I'd propose to create `Jinjava` in a separate `Producer` (e.g. like it is done [here](https://github.com/eclipse-ee4j/krazo/blob/master/ext/thymeleaf/src/main/java/org/eclipse/krazo/ext/thymeleaf/DefaultTemplateEngineProducer.java))? 

This way an user can easily overwrite the configuration by using `@Specializes` and doesn't need to extend the whole `ViewEngine`